apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/AbstractCleanupJob.java

```diff
@@ -25,12 +22,12 @@ public abstract class AbstractCleanupJob implements CleanupJob {
 
   /** template for delete query. */
   private static final String DELETE_QUERY_TEMPLATE =
-      "delete from ${tableName} t where t.${parentTableKey} in ("
+      "delete from ${parentTableName} t where t.${parentTableKey} in ( "
           + "  select ${parentTableKey} "
           + "  from ${parentTableName} "
-          + "  where last_updated < '${cutoffDate}' "
-          + "  limit ${limit}"
-          + ")";
+          + "  where last_updated between "
```

Comment:
the sub query seems result in a relatively constant set of parent table keys, hope postgres optimize it (do it only once)